src/main/java/com/khadri/jakarta/product/servlet/modify/ProductModifySearchPageServlet.java

```diff
@@ -36,20 +36,38 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 		StringBuffer sb = new StringBuffer();
 
 		sb.append("<html>");
+		sb.append("<head>");
+		sb.append("<title>Modify Product Details</title>");
+		sb.append("<script type='text/javascript'>");
+		sb.append("function validateForm() {");
+		sb.append("var id = document.getElementById('productId').value;");
+		sb.append("if (id == '') {");
+		sb.append("alert('Please enter a Product ID ..!');");
+		sb.append("return false;");
+		sb.append("}");
+		sb.append("if(isNaN(productIdComponent)) { ");
```

Comment:
isNaN or isName?

Reply:
isNanN is method when productId given a alphabet It gives one alert